tools/bundle-builder/docs/index.js

```diff
@@ -249,6 +249,8 @@ function buildSite_getData() {
     nav.push({
       name: componentData.name,
       component: componentName,
+      hide: componentData.hide,
+      fastLoad: componentData.fastLoad,
```

Comment:
Nav data as passed to the template is its own structure, so we have to copy these properties here.